packages/expo-updates/android/src/main/java/expo/modules/updates/db/dao/UpdateDao.kt

```diff
@@ -90,11 +90,11 @@ abstract class UpdateDao {
   abstract fun _setUpdateCommitTime(id: UUID, commitTime: Date)
 
   @Transaction
-  open fun markUpdateFinished(update: UpdateEntity, hasSkippedEmbeddedAssets: Boolean) {
+  open fun markUpdateFinished(update: UpdateEntity, isEmbedded: Boolean) {
     var statusToMark = UpdateStatus.READY
     if (update.status === UpdateStatus.DEVELOPMENT) {
       statusToMark = UpdateStatus.DEVELOPMENT
-    } else if (hasSkippedEmbeddedAssets) {
```

Comment:
hmm, this seems different. In the past it would only set this to `UpdateStatus.EMBEDDED` if there were skipped assets? So the equivalent behavior now that we always download all the assets is `UpdateStatus.READY`.

Reply:
It is different -- and it seems more correct to me to always distinguish between EMBEDDED and READY status. I'll do another pass over the code, and also check that updates E2E doesn't catch any issues. As it was, the code would have two possible different statuses for embedded bundles, which doesn't seem right.

Reply:
If there are any issues, we can just remove the EMBEDDED status entirely.

Reply:
The comment on the EMBEDDED status specifically states that it is for partially-downloaded embedded updates, so I'm tempted to just remove it:
```
  /**
   * The update has been partially loaded (copied) from its location embedded in the app bundle, but
   * not all assets have been copied successfully. The update may be able to be launched directly
   * from its embedded location unless a new binary version with a new embedded update has been
   * installed.
   */
```

Reply:
For example, in`packages/expo-updates/android/src/main/java/expo/modules/updates/launcher/DatabaseLauncher.kt` it seems that there is code that expects embedded updates to have `EMBEDDED` status. See lines 131 and following.

Reply:
Ugh, yeah, hard to know what was expected here since not all embedded updates were marked as embedded. In fact, most probably weren't.

Reply:
I think the safest thing to do is never set `EMBEDDED` but keep the status since old database entries will have it.

Reply:
Agreed, done.